app/src/main/java/no/usn/mob3000/ui/screens/auth/ForgotPasswordScreen.kt

```diff
@@ -91,14 +93,15 @@ fun ForgotPasswordScreen(
                     onValueChange = { email = it },
                     placeholder = { Text(stringResource(R.string.auth_login_email)) },
                     label = { Text(stringResource(R.string.auth_login_email_placeholder)) },
+                    textStyle = TextStyle(color = MaterialTheme.colorScheme.onBackground),
```

Comment:
[See this comment](https://github.com/frigvid/mob3000-group11/pull/27/files#r1847123849).